client/src/Organisms/ContestFilterHarmonica/ContestFilterToggler.js

```diff
@@ -2,19 +2,24 @@ import { GoTriangleDown, GoTriangleUp } from 'react-icons/go';
 
 import ContestFilterTogglerStyled from './ContestFilterToggleStyled';
 import PropTypes from 'prop-types';
+import useMediaQuery from '../../Hooks/useMediaQuery';
 
 const ContestFilterToggler = ({ onClick, toggle }) => {
   return (
     <ContestFilterTogglerStyled onClick={onClick}>
-      {toggle ? <GoTriangleUp /> : <GoTriangleDown />}
+      {useMediaQuery('(min-width:800px)') && !toggle ? null : toggle ? (
```

Comment:
Maybe its a other way to write this nested expression ? This way is a bit hard to read